components/input.css

```diff
@@ -82,9 +92,63 @@
     font-size: 1rem;
 }
 
+.input-container {
+    color: var(--font-color);
+    font-family: Helvetica, Arial, sans-serif;
+    font-size: 1rem;
+    line-height: 1.25rem;
+    margin: 0 1.25rem 0.6rem;
+    width: 14rem;
+}
+
+.input-container label {
+    display: block;
+    margin-bottom: 5px;
+}
+
+.input-container input {
+    background: var(--white);
+    border: 1px solid var(--text-color);
+    float: left;
+    font-size: 0.8rem;
+    height: 2rem;
+    margin: 0;
+    padding: 0 0 0 1rem;
+    width: 100%;
+}
+
+.input-container .error-message {
+    color: #cc0033;
+    display: inline-block;
+    font-size: 1rem;
+    line-height: 1rem;
+    margin: 5px 0 0;
+}
+
+.input-container .input-group .error-message {
+    display: none;
+}
+
+
+/* Error Styling */
+
+.input-container .error label {
+    color: var(--danger-color);
+}
+
+.input-container .error input[type=text] {
+    background-color: var(--white);
+    border: 1px solid var(--danger-color);
+    outline: none;
+}
+
+.input-container .error .error-message {
+    display: inline-block;
+}
+
 /* css for mobile responsive */
-@media only screen and (max-width: 600px){
-    .input-field{
+@media only screen and (max-width: 600px) {
+    .input-field {
         padding: 0.5rem;
         margin: 0.25rem;
```

Comment:
Suggestion : you can use px here for less than 1rem values

Reply:
I used to use rem.